coordinator/setup.py

```diff
@@ -288,7 +288,7 @@ def parsed_packages():
     name = os.environ.get("package_name", "gs-coordinator")
     if name == "gs-coordinator":
         return find_packages(".")
-    return ["foo"]
+    return ["graphscope_runtime"]
```

Comment:
Also change the  module name 'foo' in coordinator directory

https://github.com/alibaba/GraphScope/tree/main/coordinator/foo

Reply:
Already renamed.